File: PTA/Advanced_Level/1001.cpp
#include<bits/stdc++.h>
using namespace std;


int main() {
	int a, b;
		cin >> a >> b;
	string str = to_string(a + b);
	string res;

	for(int i = str.size() - 1, j = 0; i >= 0; --i) {
		res = str[i] + res;
		if(++j % 3 == 0 && i - 1 >= 0 && str[i - 1] != '-')
			res = ',' + res;
	}
	cout << res << endl;
}

File: PTA/Advanced_Level/1001.rs
use std::io::stdin;

fn main() {
    let mut input: String = String::new();
    stdin().read_line(&mut input).unwrap();
    let v: Vec<i32> = input.trim().split(' ').map(|s| s.parse::<i32>().unwrap()).collect();
    //println!("{:?}", v);
    let num = (v[0] + v[1]).to_string();
    //println!("{}", num);
    let mut res = String::new();
    let mut j: usize = 0;
    for x in num.chars().rev() {
        if j % 3 == 0 && x != '-' && j != 0 {
            res = String::from(',') + res.as_str();
        }
        j += 1;
        res = String::from(x) + res.as_str();
    }
    println!("{}", res);
}

